Atomic/util.py

```diff
@@ -254,3 +253,19 @@ def urllib3_disable_warnings():
             # Except only call disable-warnings if it exists
             if hasattr(urllib3, 'disable_warnings'):
                 urllib3.disable_warnings()
+
+
+def skopeo(image):
```

Comment:
how does this translates a provided image called `busybox` to docker's `DefaultRegistries[0-N]/busybox`?